changelogger/config/initializers/wrap_parameters.rb

```diff
@@ -0,0 +1,14 @@
+# Be sure to restart your server when you modify this file.
+
+# This file contains settings for ActionController::ParamsWrapper which
+# is enabled by default.
+
+# Enable parameter wrapping for JSON. You can disable this by setting :format to an empty array.
+ActiveSupport.on_load(:action_controller) do
+  wrap_parameters format: [:json]
+end
+
+# To enable root element in JSON for ActiveRecord objects.
+# ActiveSupport.on_load(:active_record) do
+#   self.include_root_in_json = true
+# end
```

Comment:
I had add back wrapped params because when we removed it, rails overwrote the `action` key to be the rails action and there was no way to get the original one from the payload. So, now we have a `payload` method that returns `params["webhook"]` which is the only time we access params. Everywhere else we use the `payload` method.